ale_linters/powershell/psscriptanalyzer.vim

```diff
@@ -0,0 +1,154 @@
+" Author: Jesse Harris - https://github.com/zigford
+" Description: This file adds support for lintng powershell scripts
+"   using the PSScriptAnalyzer module.
+
+" let g:ale_powershell_psscriptanalyzer_exclusions =
+" \ 'PSAvoidUsingWriteHost,PSAvoidGlobalVars'
+call ale#Set('powershell_psscriptanalyzer_exclusions',
+\ get(g:,
+\ 'ale_linters_powershell_psscriptanalyzer_exclusions', ''))
+call ale#Set('powershell_psscriptanalyzer_executable', 'pwsh')
+call ale#Set('powershell_psscriptanalyzer_module',
+\ 'psscriptanalyzer')
+
+" GetExecutable {{{
```

Comment:
Get rid of these `{{{` and `}}}` comments. You can use the indentation levels for folding instead.